plugin.video.hanssettings/resources/lib/hanssettings.py

```diff
@@ -0,0 +1,94 @@
+'''
+    resources.lib.hanssettings
+    ~~~~~~~~~~~~~~~~~
+
+    An XBMC addon for watching HansSettings
+   
+    :license: GPLv3, see LICENSE.txt for more details.
+
+    Uitzendinggemist (NPO) = Made by Bas Magre (Opvolger)    
+    based on: https://github.com/jbeluch/plugin.video.documentary.net
+
+'''
+try:
+    # For Python 3.0 and later
+    from urllib.request import urlopen, Request    
+except ImportError:
+    # Fall back to Python 2's urllib2
+    from urllib2 import urlopen, Request    
+
+import re ,time ,json
+from datetime import datetime
+PY3 = False
+
+# hierin geen logica van kodi, zodat dit los testbaar is.
+
+class HansSettings:
+        #
+        # Init
+        #
+        def __init__(self, py3):
+            self.PY3 = py3
+
+        def get_dataoverzicht(self):
+            return self.get_datafromfilegithub('bouquets.tv')
+
+        def get_datafromfilegithub(self, file):
+            return self.get_datafromurl('https://raw.githubusercontent.com/haroo/HansSettings/master/e2_hanssettings_kabelNL/' + file)
```

Comment:
This is not allowed in our repository. If you need to provide access to random files you have to include the file itself with the addon. We need to check exactly which kind of content the user is accessing.

Reply:
I had not thought about that. 
I do understand why you (kodi) do not want that.
Then I have to update regularly. Because this list is updated monthly

Reply:
There is no problem with that, we can handle the updates. In fact we are working on something to make the updates automatic and save devs the hassle of having to handle forks of this repo. In case you are interested: https://github.com/romanvm/kodi-addon-submitter

This is still WIP though.

Reply:
I'll take a look at it if I have some time again. Will first adjust the addon. Have some review points to process :)

Reply:
files are now in the plugin.